Plots/Satellites/NCL_sat_3.py

```diff
@@ -0,0 +1,155 @@
+                    va='center', ha='left',
+                    transform=ccrs.PlateCarree())
+
+    if loc == 'top':
+        for lon, lat in coords:
+            ax.text(lon, lat,
+                    '{0}\N{DEGREE SIGN} W '.format(-lon),
+                    va='bottom', ha='center',
+                    transform=ccrs.PlateCarree())
+
+    if loc == 'bottom':
+        for lon, lat in coords:
+            ax.text(lon, lat,
+                    '{0}\N{DEGREE SIGN} W '.format(-lon),
+                    va='top', ha='center',
+                    transform=ccrs.PlateCarree())
+
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and
+# load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/h_avg_Y0191_D000.00.nc'),
+                     decode_times=False)
+
+# Extract a slice of the data
+t = ds.T.isel(time=0, z_t=0)
+
+###############################################################################
+# Plot:
+
+plt.figure(figsize=(8, 8))
+
+# Create an axis with an orthographic projection
+ax = plt.axes(projection=ccrs.Orthographic(central_longitude=-35,
+                                           central_latitude=60), anchor='C')
+
+# Set extent of map
+ax.set_extent((-80, -10, 30, 80), crs=ccrs.PlateCarree())
+
+# Add natural feature to map
+ax.coastlines(resolution='110m')
+ax.add_feature(cfeature.LAND,
+               facecolor='lightgray', zorder=3)
+ax.add_feature(cfeature.COASTLINE,
+               linewidth=0.2, zorder=3)
+ax.add_feature(cfeature.LAKES,
+               edgecolor='black', linewidth=0.2, facecolor='white', zorder=4)
+
+# Plot gridlines
+gl = ax.gridlines(color='black', linewidth=0.2, zorder=2)
+
+# Set frequency of gridlines in the x and y directions
+gl.xlocator = mticker.FixedLocator(np.arange(-180, 180, 15))
+gl.ylocator = mticker.FixedLocator(np.arange(-90, 90, 15))
+
+# plot filled contour data
+heatmap = t.plot.contourf(ax=ax,
+                          transform=ccrs.PlateCarree(),
+                          levels=80,
+                          vmin=-1.5, vmax=28.5,
+                          cmap='RdGy',
+                          add_colorbar=False,
+                          zorder=1)
+
+# Create colorbar
+cbar = plt.colorbar(heatmap,
+                    orientation='horizontal',
+                    extendfrac=[0, .1],
+                    extend='max',
+                    extendrect=True,
+                    pad=0.05,
+                    shrink=.75,
+                    aspect=14,
+                    ticks=np.arange(-1.5, 31.5, 3)
+                    )
+
+# Get rid of black outline on colorbar
+cbar.outline.set_visible(False)
+
+# Set plot titles
+main = r"$\bf{Example}$" + " " + r"$\bf{of}$" + " " + r"$\bf{Zooming}$" + \
+       " " + r"$\bf{a}$" + " " + r"$\bf{Sat}$" + " " + r"$\bf{Projection}$"
+left = 'Potential Temperature'
+right = 'Celsius'
```

Comment:
Instead of hardcoding these values, you may be able to use `t.long_name` and `t.units`. NetCDF files have variable names and units built in as part of the metadata and this is how xarray lets you access it.